lib/please_cli/please_cli/config.py

```diff
@@ -133,11 +137,15 @@
                 'heroku_app': 'releng-staging-archiver',
                 'heroku_dyno_type': 'web',
                 'url': 'https://archiver.staging.mozilla-releng.net',
+                # TODO: switch to SSL Endpoint
```

Comment:
@dividehex can we switch to SSL Endpoint for ``releng-staging-archiver`` heroku app?